molecule/config.py

```diff
@@ -102,6 +103,20 @@ def after_init(self):
     def write(self):
         util.write_file(self.config_file, util.safe_dump(self.config))
 
+    @property
+    def ansible_version(self):
+        """Return current version of ansible."""
+        return pkg_resources.get_distribution("ansible").version
+
+    @property
+    def ansible_collections_path(self):
+        """Return collection path variable for current version of Ansible."""
+        # https://github.com/ansible/ansible/pull/70007
+        if self.ansible_version >= "2.10":
```

Comment:
That is not how you should compare version numbers. Better use `LooseVersion` (`from distutils.version import LooseVersion`) to compare version numbers.

Reply:
Fixed using more modern version from packaging instead of outdated distutils.

Reply:
If you already have `packaging` as a requirement, that's better. If you don't, you need to add it as a requirement.

Reply:
You don't seem to have it in your dependencies (https://github.com/ansible-community/molecule/blob/master/pyproject.toml#L3-L7), so you need to add it (https://pypi.org/project/packaging/).